tests/ConditionTest.php

```diff
@@ -32,21 +32,22 @@ public function testBasicDiscrimination()
 
         $m->addCondition('gender', 'M');
 
-        $this->assertSame(1, count($m->conditions));
+        $this->assertEquals(1, count($m->scope()->getNestedConditions()));
```

Comment:
use `assertSame`, `assertEquals` checks types weakly (and already prohibited by CS in other repos)

Reply:
Done